internal/server/server.go

```diff
@@ -18,13 +18,19 @@ type GrafanaResponseHeaders struct {
 	User string
 }
 
+type GrafanaClaimsMap struct {
```

Comment:
nit:  This isnt a map.  `GrafanaClaimsConfig`

Reply:
done